evap/evaluation/tests/test_views.py

```diff
@@ -94,15 +94,36 @@ class TestFAQView(WebTestWith200Check):
 class TestContactEmail(WebTest):
     csrf_checks = False
 
+    @override_settings(ALLOW_ANONYMOUS_FEEDBACK_MESSAGES=True)
     def test_sends_mail(self):
         user = baker.make(UserProfile, email="user@institution.example.com")
+        # normal email
         self.app.post(
             "/contact",
-            params={"message": "feedback message", "title": "some title", "sender_email": "unique@mail.de"},
+            params={"message": "feedback message", "title": "some title", "anonymous": "false"},
             user=user,
         )
-        self.assertEqual(len(mail.outbox), 1)
-        self.assertTrue(mail.outbox[0].reply_to == ["user@institution.example.com"])
+        # anonymous email
+        self.app.post(
+            "/contact",
```

Comment:
Would it make sense to have this as `self.url`, maybe from a `reverse()` call? (It's not three times in the tests here).